### ISSUE TITLE
No sound on Gradient Descent Video

### ISSUE THREAD

**Issue by vuvuzella** [udacity/robotics-nanodegree-issues#208]
Intro to Neural Networks - 24. Gradient Descent
Video does not have sound as of this posting.

**Comment by sahiljuneja** [comment]
This should be fixed now. Thanks for reporting!
